Packs/QualysVulnerabilityManagement/Playbooks/qualys-pc-scan-launch-and-fetch.yml

```diff
@@ -0,0 +1,338 @@
+id: qualys-pc-scan-launch-and-fetch
```

Comment:
please follow the [playbook conventions](https://xsoar.pan.dev/docs/playbooks/playbook-conventions)

Reply:
Changed it.